src/ext/property-definition/trendlines-definition.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-param-reassign */
+import { trendlinesService } from 'qlik-chart-modules';
+
+const onGlobalChange = (properties) => {
+  if (properties.qHyperCubeDef?.qMeasures?.length >= 2) {
+    properties.qHyperCubeDef.qMeasures[0].qTrendLines = [];
+    properties.qHyperCubeDef.qMeasures[1].qTrendLines = [];
+    properties.trendLines?.forEach((qTrendLine) => {
+      if (qTrendLine.flipXY) {
+        qTrendLine.qXColIx = 2;
+        properties.qHyperCubeDef.qMeasures[0].qTrendLines.push(qTrendLine);
+      } else {
+        qTrendLine.qXColIx = 1;
+        properties.qHyperCubeDef.qMeasures[1].qTrendLines.push(qTrendLine);
+      }
+    });
+  }
+};
+
+export default function getDefinition(env) {
+  const { flags, translator } = env;
+  const trendlines = trendlinesService.propertyPanelProperties({ translator });
```

Comment:
I guess we'd like to set the label for the "Add" button dynamically through here? Right now I saw it said "Add trend line"...